include/swift/SIL/SILInstruction.h

```diff
@@ -5881,6 +5881,66 @@ class ObjCSuperMethodInst
       : UnaryInstructionBase(DebugLoc, Operand, Ty, Member) {}
 };
 
+// This is either the extracted method or the adjusted this ptr that
+// is the result of a CXXVirtualMethodInst.
```

Comment:
Nitpick: please make this into an actual doc comment for CXXVirtualMethodResult.